CHANGELOG.markdown

```diff
@@ -33,6 +33,8 @@
 
 * Add `Semigroup` and `Monoid` instances for `Indexing`.
 
+* Added `Cons` and `Snoc` instances for `Control.Applicative.ZipList`
```

Comment:
4.16 is already out—put this changelog entry under a new header (use `next` as the version).

Reply:
Done